libs/langchain/langchain/document_loaders/readthedocs.py

```diff
@@ -3,11 +3,13 @@
 
 from langchain.docstore.document import Document
 from langchain.document_loaders.base import BaseLoader
+import re
 
 
-class ReadTheDocsLoader(BaseLoader):
+class ReadDocLoader(BaseLoader):
```

Comment:
Renaming the loader is a breaking change (will break existing users code)

Reply:
Apologies for this oversight, I was testing the original vs my implementation which is why I had changed the name. This is taken care of now, working on the unit tests